alicloud/table_alicloud_vpc_eip.go

```diff
@@ -275,9 +282,9 @@ func getVpcEipAka(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateDat
 	commonColumnData := commonData.(*alicloudCommonColumnData)
 	accountID := commonColumnData.AccountID
 
-	akas := []string{"acs:vpc:" + data.RegionId + ":" + accountID + ":eip/" + data.AllocationId}
+	arn := "acs:vpc:" + data.RegionId + ":" + accountID + ":eip/" + data.AllocationId
```

Comment:
format should be arn:acs:vpc:cn-hangzhou:123456789xxxxxxx:eip/eip-xxxxxxxxxxxxxx
https://www.alibabacloud.com/help/doc-detail/172061.htm#title-q22-kas-ids